python/openvino_tokenizers/convert_tokenizer.py

```diff
@@ -99,11 +99,14 @@ def convert_tokenizer(
 
     if ov_tokenizers is None:
         raise OVTypeError(f"Tokenizer type is not supported: {type(tokenizer_object)}")
+    
+    ov_tokenizer = ov_tokenizers[0] if isinstance(ov_tokenizers, tuple) else ov_tokenizers
+    make_combine_segments_stateful(ov_tokenizer, add_special_tokens)
 
     if isinstance(ov_tokenizers, tuple):
         return (
             change_outputs_type(ov_tokenizers[0], tokenizer_output_type),
             change_inputs_type(ov_tokenizers[1], detokenizer_input_type),
         )
-
-    return change_outputs_type(ov_tokenizers, tokenizer_output_type)
+    else:
+        return change_outputs_type(ov_tokenizers, tokenizer_output_type)
```

Comment:
This code seems equivalent, why change it?

Reply:
done